src/dev/src/app/quick-list/quick-list.demo.html

```diff
@@ -0,0 +1,30 @@
+<h2>Quick List</h2>
+<div class="clr-example">
+  <div class="main-container">
+    <div class="clr-row">
+      <clr-quick-list class="clr-col-6 scroll" [clrAllValues]="possibleActivityResults" [(clrValues)]="selectedActivityResults"
```

Comment:
As requested, please add a real life example inside a form with at least on normal input field with label/value. I don't think this will work that easy, right now.
I guess this will be used alongside other form elements. So I want to check if it fits in this layout.